cime_config/machines/config_machines.xml

```diff
@@ -1136,6 +1136,7 @@
     <CCSM_CPRNC>/lcrc/group/e3sm/tools/cprnc/cprnc</CCSM_CPRNC>
     <GMAKE_J>8</GMAKE_J>
     <TESTS>e3sm_integration</TESTS>
+    <NTEST_PARALLEL_JOBS>8</NTEST_PARALLEL_JOBS>
```

Comment:
The two login nodes have 128 cores. At least half of a node should be available to other tasks while jenkins tests are building. Non-jenkins tests can be launched from compute nodes.